pkgs/applications/graphics/drawio/default.nix

```diff
@@ -65,6 +71,11 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
+  '' + lib.optionalString stdenv.isDarwin ''
+    mkdir -p $out/{Applications,bin}
+    mv dist/mac*/draw.io.app $out/Applications
+    makeWrapper $out/Applications/draw.io.app/Contents/MacOS/draw.io $out/bin/drawio
```

Comment:
`makeWrapper` with no arguments seems a bit strange — why not just a symlink?

Reply:
`$out/bin/drawio` will then be unable to find files in the macOS bundle by relative paths.

```
[1021/144723.293589:FATAL:electron_main_delegate_mac.mm(71)] Unable to find helper app
```

Reply:
Wow, that's horrible.  It could maybe use a comment.

Reply:
Copied a comment from `utm`.